Updated to align with the ISO standards

lalc opened the pull request:
Please review

jose-gataca commented on interface-specs/data-agreement-schema/v1/data-agreement-schema.json:
```diff
@@ -1,57 +1,52 @@
-    "dpia_done": true,
     "dpia_date": "2021-05-08T08:41:59+0000",
-    "dpia_verification_url": "https://org.com/dpia_results.html"
+    "dpia_summary_url": "https://org.com/dpia_results.html"
```
Is this better than updating on the status of a dpia?
like "dpia_status": "NOT_STARTED | EXECUTED | APPROVED | TO REWORK" or similar

jose-gataca replied:
If it's not needed, why not only display a reference to the summary, which would also include the date? Simplify the object for a URI reference

lalc replied:
Lets discuss this during the call

jose-gataca commented on interface-specs/data-agreement-schema/v1/data-agreement-schema.json:
```diff
@@ -1,57 +1,52 @@
+  "event":[{
+    "time-stamp": "2021-05-08T08:41:59+0000",
+    "principle-did": "did:mydata:1:<sender_did_value>",
+    "state": "prepared",
```
Can you elaborate on the state posible values?

lalc replied:
I have corrected the possible values based on the DA workflow. The possible values are <Defintion/Prepration/Capture/Proof>

jose-gataca reviewed (changes requested):
I would be missing a reference somewhere to the pii_subject.

Also, I would suggest using linked data proofs or similar: https://w3c-ccg.github.io/ld-proofs/#proof-algorithm. I think that would ease the adaptation and integration, as well as understanding the cryptographic suites and protocols to be able to perform those signatures.

Finally, I am not sure if I understand what should happen with the PRs 6 and 7 of both Jan and me.

lalc commented:
> I would be missing a reference somewhere to the pii_subject.

It's as in the DA spec. pii-subject comes in during the capture phase. 
> 
> Also, I would suggest using linked data proofs or similar: https://w3c-ccg.github.io/ld-proofs/#proof-algorithm. I think that would ease the adaptation and integration, as well as understanding the cryptographic suites and protocols to be able to perform those signatures.

Thanks. Will take care of this in the signature. Good point.

> 
> Finally, I am not sure if I understand what should happen with the PRs 6 and 7 of both Jan and me.

Your PR for adding vui-implementation can remain once you fix as discussed.

When it comes to updated the data-agreement schema, this will hold and my assumption was you have raised the PR for discussions.

JanLin commented:
Recommend make some changes to the schema.

1. Change field "data_retention_period_in_months" to "data_retention". The field name should include format of data. This shall be done descriptively.
2. Jurisdiction example should simply be country. Remove Stockholm so it only reads "Sweden".
3. The field "geographic_restriction" has example "Europe, Not restricted" which makes it unclear. The field when included is automatically a restriction. Instead includes basis for restriction which may be storage_location or jurisdiction. So either "Europe" or "Sweden".
4. Add under "Name" category personal data a field indicating it is sensitive:

"attribute_sensitive": "True"

lalc commented:
> Recommend make some changes to the schema.
> 
> 1. Change field "data_retention_period_in_months" to "data_retention". The field name should include format of data. This shall be done descriptively.
Done
> 2. Jurisdiction example should simply be country. Remove Stockholm so it only reads "Sweden".
Done
> 3. The field "geographic_restriction" has example "Europe, Not restricted" which makes it unclear. The field when included is automatically a restriction. Instead includes basis for restriction which may be storage_location or jurisdiction. So either "Europe" or "Sweden".
Done
> 4. Add under "Name" category personal data a field indicating it is sensitive:
> "attribute_sensitive": "True"
We had removed this earlier based on the discussion. Added this back now.
